recipe/meta.yaml

```diff
@@ -62,3 +80,7 @@ extra:
   recipe-maintainers:
     - ellisonbg
     - jakevdp
+    - joelostblom
+    - jonmmease
+    - binste
+    - mattijn
```

Comment:
I added everyone listed on PyPI as a package maintainer here so that we get tagged in the PRs made to this repo. Let me know if you want you name removed